opennlp-tools/src/test/java/opennlp/tools/namefind/TokenNameFinderEvaluatorTest.java

```diff
@@ -26,20 +26,34 @@
 import opennlp.tools.cmdline.namefind.NameEvaluationErrorListener;
 import opennlp.tools.util.Span;
 
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
 /**
  * This is the test class for {@link TokenNameFinderEvaluator}..
  */
 public class TokenNameFinderEvaluatorTest {
 
 
+  public TokenNameFinder mockTokenNameFinder1(Span[] ret) {
```

Comment:
Maybe we should document this mock, we could even adapt the docs from the removed dummy class.

Reply:
Thanks for the comments! I revised the code correspondingly based on your advice :+1: